.appveyor.yml

```diff
@@ -0,0 +1,28 @@
+build: false
+
+environment:
+  matrix:
+    - PYTHON: "C:\\Miniconda36-x64"
+      PY: 3.7
+
+init:
+  - "ECHO %PYTHON_VERSION% %MINICONDA%"
```

Comment:
I see the following in the build:
```
ECHO is off.
```

I couldn't find a setting to enable `ECHO` either, but again, I am new to AppVeyor.

And should this be `echo` instead?

Reply:
Windows is case insensitive so it does not matter if it is `ECHO` or `echo` but yeah, that does not work to debug the env vars :-/

But we don't really need that b/c we have the full output of the packages installed down the log.